hash/nes.xml

```diff
@@ -56703,7 +56701,7 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 	<software name="fengshnb" supported="no">
 		<description>Feng Shen Bang (Chi)</description>
```

Comment:
This should be "Fēngshénbǎng"

Reply:
This is a historical novel. The spacing here seems to be standard. Reference: https://en.wikipedia.org/wiki/Fengshen_Yanyi